NEWS.rst

```diff
@@ -0,0 +1,6 @@
+3.1.1.dev18+g17f6470 (2017-05-23)
+==================
+
```

Comment:
I think this doesn't show anything because of the doc change correct? Could you add a few dummy entries for bugfix, features, etc so we can spot any problems with the template?